locales/en/validation.json

```diff
@@ -22,6 +22,7 @@
   "default": "is required",
   "default_with_label": "{{label}} is required",
   "generic": "something went wrong",
+  "prisonNumber_with_label": "{{label}} must be of the format A1234AB",
```

Comment:
Just had a quick look at the Design System for [error messages](https://design-system.service.gov.uk/patterns/national-insurance-numbers/#error-messages) and I wonder if we should add a hint to the field and simplify the error message:

**Prison number**
The person's identifier within the prison system. For example, A1234AB.
_Enter a prison number in the correct format_
[.....................]

Reply:
I don't think we need to add "The person's identifier within the prison system." It's working fine without this message, bit overkill otherwise. 

Content should be as previous:

> [Title] Prison number
> [Hint] For example, 'A1234AB'
> 
> Then can we have the error message display as:
> [Error] Enter a prison number in the correct format

Reply:
Yep, think you're right. I wasn't sure on that either actually!

I also only just spotted you said the same thing a few days ago 🤦  Great minds 😉